search/includes/classes/queue/class-cron.php

```diff
@@ -46,7 +46,7 @@ class Cron {
 	/**
 	 * The maximum number if processor jobs allowed at one time
 	 */
-	const MAX_PROCESSOR_JOB_COUNT = 5;
+	const MAX_PROCESSOR_JOB_COUNT = 3;
```

Comment:
Just wondering, but will this have much of an impact of it evaluates to 2.5 usually? https://github.com/Automattic/vip-go-mu-plugins/blob/90ae5a0b46c0231fce4384075f314a5cb20f7613/search/includes/classes/queue/class-cron.php#L86-L89

Reply:
I spot-checked some heavy hitters, they have much bigger concurrency, which kind of adds up to the problem.